modules/roles_profiles/manifests/profiles/virtual_drivers.pp

```diff
@@ -3,27 +3,25 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 class roles_profiles::profiles::virtual_drivers {
+  case $facts['os']['name'] {
+    'Windows': {
+      $flags    = '-s -k 30570681-0a8b-46e5-8cb2-d835f43af0c5'
+      $vac_dir  = lookup('windows.dir.vac')
+      $version  = lookup('win-worker.vac.version')
+      $work_dir = "${vac_dir}\\vac${version}"
 
-    case $::operatingsystem {
-        'Windows': {
-
-            $flags    = '-s -k 30570681-0a8b-46e5-8cb2-d835f43af0c5'
-            $vac_dir  = lookup('windows.dir.vac')
-            $version  = lookup('win-worker.vac.version')
-            $work_dir = "${vac_dir}\\vac${version}"
-
-            class { 'win_packages::vac':
-                flags    => $flags,
-                srcloc   => lookup('windows.s3.ext_pkg_src'),
-                vac_dir  => $vac_dir,
-                version  => $version,
-                work_dir => $work_dir
-            }
-            # Bug List
-            # https://bugzilla.mozilla.org/show_bug.cgi?id=1656286
-        }
-        default: {
-            fail("${::operatingsystem} not supported")
-        }
+      class { 'win_packages::vac':
```

Comment:
No change needed. Just want to flag this because it is an old piece of software. Hopefully it will just work on Win 11 like Win 10, we will find out during testing.

Reply:
Looks like [it might](https://vac.muzychenko.net/en/history.htm) with 4.67.

Reply:
Let's go ahead an update to the latest for win 11.  We can take a look at that next week.